src/main/resources/templates/fragments/exam-file.html

```diff
@@ -2,8 +2,9 @@
 <html lang="en">
 <body>
 <div>
-    <p th:text="${examFile.getSubjectName()}"></p>
-    <p th:text="${examFile.getFileName()}"></p>
+    <a th:href="@{|/download/${examFile.getUrl}|}">
+        <span th:text="${examFile.getSubjectName()}"></span>
```

Comment:
Can't you set the text directly on the <a> tag?

Reply:
Its In the ```<a>``` now